apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/Record.java

```diff
@@ -51,4 +83,17 @@ public String key() {
   public Map<String, Object> fields() {
     return fields;
   }
+
+  public int sizeEstimateBytes() {
+    if (memoizedSize) {
+      return sizeEstimateBytes;
+    }
+    return calculateSizeEstimate(FieldsAdapter.create());
+  }
+
+  private int calculateSizeEstimate(FieldsAdapter adapter) {
```

Comment:
Can we inject this calculation to the place where we create Record, for instance:
in com.apollographql.android.cache.normalized.sql.SqlStore#cursorToRecord

there we have already `String jsonOfFields = cursor.getString(2);` so can we take size of `jsonOfFields` as size for record?